src/api/app/controllers/application_controller.rb

```diff
@@ -569,6 +569,17 @@ def set_response_format_to_xml
     request.format = :xml if request.format == :html
   end
 
+  def extract_user_public
```

Comment:
@mdeniz yeah I like that style much better too, but doesn't it still need to include the `return true`? Like this?
```ruby
def extract_user_public
  if ::Configuration.anonymous
    load_nobody
    return true
  else
    logger.error 'No public access is configured'
    render_error( message: 'No public access is configured', status: 401 )
  end
end
```